jekyll/faq.md

```diff
@@ -39,10 +39,12 @@ Nim.
 
 ## How stable is Nim?
 
-The compiler is in development and some important features are still missing.
-However, the compiler is quite stable already: It is able to compile itself
-and a substantial body of other code. Until version 1.0.0 is released,
-minor incompatibilities with older versions of the compiler will be introduced.
+The compiler and stdlib are actively under development and have a large suite
+of test cases to verify ongoing stability. Regular releases are posted every
```

Comment:
"The compiler and stdlib are actively under development and have a suite of test cases to ensure ongoing stability."